sources/hubspot/settings.py

```diff
@@ -5,11 +5,11 @@
 STARTDATE = pendulum.datetime(year=2000, month=1, day=1)
 
 CRM_CONTACTS_ENDPOINT = (
-    "/crm/v3/objects/contacts?associations=deals,products,tickets,quotes&limit=100"
+    "/crm/v3/objects/contacts?properties={properties}associations=deals,products,tickets,quotes&limit=100"
```

Comment:
without ampersand in the url, you are producing the query (assoming props a,b) "properties=a,bassociations=deals" which will not be handled correctly